.travis.yml

```diff
@@ -27,16 +27,14 @@ matrix:
     - os: linux
       env: PROJECT_DIR=examples/Snappy TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
     - os: linux
-      env: PROJECT_DIR=examples/Snappy TOOLCHAIN=analyze
-    - os: linux
+    # Static analysis finds bugs in Snappy.
+    #   env: PROJECT_DIR=examples/Snappy TOOLCHAIN=analyze
+    # - os: linux
       env: PROJECT_DIR=examples/Snappy TOOLCHAIN=sanitize-address
     - os: linux
       env: PROJECT_DIR=examples/Snappy TOOLCHAIN=sanitize-leak
-
-    # FIXME: https://github.com/ruslo/hunter/issues/718#issuecomment-290610583
-    # - os: linux
-    #   env: PROJECT_DIR=examples/Snappy TOOLCHAIN=sanitize-thread
-
+    - os: linux
```

Comment:
Currently pkg.snappy contains no valid code so this is `RED` -> `RED` state. The only changes that can be merge is "disable something". After merge I will test pull request again and if everything fine I will update pkg.snappy. It will become `GREEN`. Then you can enable toolchain and we can see that state doesn't change to `RED`. Also you can test it in your fork yourself, you just need to register your Hunter fork in Travis.